### PR TITLE
feat-261: update home screen light mode

### PR DESCRIPTION
**Pull request by mgibson-casper**
### 🔥 Summary
https://github.com/casper-network/casper-blockexplorer-frontend/issues/261

### 📹 Video

Screenshot 2023-04-18 at 9 48 10 PM

### 😤 Problem / Goals
-

### 🤓 Solution
-

### 🗒️ Additional Notes
-

### CODE REVIEW

File: app/src/components/layout/Home/HomeComponents.styled.ts [hunk 1]
```diff
@@ -48,29 +49,28 @@ export const PageLink = styled(Link)`
...
-  font-weight: ${fontWeight.medium};
+  color: ${colors.black};
   font-size: clamp(0.9rem, 1.25vw, 1rem);
 `;
 
 export const Card = styled.section`
   background: ${colors.white};
   padding: 0 2rem;
-  border: 0.063rem solid ${colors.mediumSupporting};
+  border: ${pxToRem(4)} solid #4589f6;
```
Comment (DorianKwan) [inline app/src/components/layout/Home/HomeComponents.styled.ts:73]: This border seems quite large.  Have the new designs required this, I don't see any details in the ticket

**Review by DorianKwan — COMMENTED** [review]
I like the changes made to the icons here -- we should make them to the casper UI kit as well when we shift over

File: app/src/components/layout/Home/HomeComponents.styled.ts (hunk 1 above)
Comment (mgibson-casper) [reply app/src/components/layout/Home/HomeComponents.styled.ts:73]: Feel free to check out the updated designs here if you'd like: https://www.figma.com/file/GS7thHuVvP72ZrnBkWrVAp/block-explorer?node-id=1102-1795&t=2wKHdBxEqRqzDyI3-0

Comment (mgibson-casper) [reply app/src/components/layout/Home/HomeComponents.styled.ts:73]: they're part of the main epic I'm working on: https://github.com/casper-network/casper-blockexplorer-frontend/issues/258

**Review by valfaro123 — APPROVED** [review]

File: app/src/components/layout/Home/HomeComponents.styled.ts (hunk 1 above)
Comment (DorianKwan) [reply app/src/components/layout/Home/HomeComponents.styled.ts:73]: Jeez, it's a completely different app 😅

**Review by DorianKwan — APPROVED** [review]

**Pull request merged**